language_detector/main.py

```diff
@@ -1,9 +1,20 @@
+# -*- coding: utf-8 -*-
 """This is the entry point of the program."""
 
 from .languages import LANGUAGES
 
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    textlist = text.split()
```

Comment:
Before splitting the string into a list, you may want to convert it to lowercase so it can be compared to the strings in the dictionaries: all lowercase (Wie != wie).